src/globus_sdk/_sphinxext.py

```diff
@@ -180,30 +183,34 @@ class ExternalDocLink(AddContentDirective):
     optional_arguments = 0
     # allow for spaces in the argument string
     final_argument_whitespace = True
-    option_spec = {"mode": directives.unchanged, "ref": directives.unchanged_required}
+    option_spec = {
+        "base_url": directives.unchanged,
+        "service": directives.unchanged,
+        "ref": directives.unchanged_required,
+    }
 
     def gen_rst(self):
         message = self.arguments[0].strip()
 
-        mode = "default"
-        if "mode" in self.options:
-            mode = self.options["mode"]
+        service: str | None = self.options.get("service")
```

Comment:
Do we have sufficient confidence that `mode` is not used across the repo?

Reply:
From a quick search I'm not seeing it used anywhere

Reply:
It was only recently introduced -- a couple of days before this PR was opened -- so unless someone introduced usage and I didn't see it, it is not yet.